### DISCUSSION TITLE
Support of zeebe 1.0.0

### DISCUSSION THREAD

**Discussion by lesnek** [camunda-community-hub/pyzeebe#177] [Q&A] [answered]
Hi,

I've seen you are ready to release pyzeebe 3.0.0, when is it planned?

We are doing in kiwi.com heavy refactor of finite state machines to zeebe and we dropped development of our client/worker for zeebe, since your worker is much more mature and better implemented.

I know it is after the fight with breaking changes, but if you need any help in pyzeebe I can provide myself, since we have high focus on zeebe in Q3.

**Comment by kbakk** [comment, marked as answer]
A release candidate of 3.0.0 has just been published on PyPI:

https://pypi.org/project/pyzeebe/#history

If you have a chance to test it, and come with feedback, that would be great!

**Reply by lesnek** [reply]
I have already tested it, works like a charm on our flows 👍 .
Async changes were surprise and our api is sync so I am glad you still kept SyncZeebeClient, maybe add it to docs.
